### PR TITLE
Osc ctxs and offscreen canvases are write only

### PR DESCRIPTION
**Pull request by caolanm**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by caolanm** [comment]
AFAICS we only write/draw *to* those offscreenCanvases/oscCtxs things and nothing reads/draws from them.

Looks to me the last read from them was removed with:

commit 1f3d5e0a3e6c90d8ba8bc8471cbcbfa94917dda9
Date:   Fri Sep 10 13:42:33 2021 +0530

    zoomframe: directly use tiles to draw zoom-frames
    
    Move the zoom frame painting to tile-section.

**Comment by mmeeks** [comment]
Sorry tilewins conflicted - but looks good otherwise - lets dung stuff out !

File: browser/src/layer/tile/CanvasTileLayer.js [hunk 1]
```diff
@@ -605,9 +605,6 @@ L.TileSectionManager = L.Class.extend({
 		this._sectionContainer.requestReDraw();
 	},
 
-	_viewReset: function () {
```
Comment (mmeeks) [inline browser/src/layer/tile/CanvasTileLayer.js:608]: There are quite a few _viewResets in sub-classes (?)

**Review by mmeeks — COMMENTED** [review]
Generally looks great - just a tad nervous about the _viewReset s- but the other bits look fun =) nice to dung that out [!]

**Comment by caolanm** [comment]
as far as I can see, before that last commit there are two _viewReset both in browser/src/layer/tile/CanvasTileLayer.js
One is TileSectionManager::_viewReset and the other is CanvasTileLayer::_viewReset

L.TileSectionManager = L.Class.extend({
...
        _viewReset: function () { // L.TileSectionManager = L.Class.extend({
        },
...
});

and 

L.CanvasTileLayer = L.Layer.extend({

        onAdd: function (map) {
                ...
                this._viewReset(); // CanvasTileLayer::_viewReset: function (e) with no e is called here

                ...
        },


        getEvents: function () {
                var events = {
                        ...
                        viewreset: this._viewReset, // CanvasTileLayer::_viewReset
                        ...
                };

                return events;
        },

        _viewReset: function (e) {
                this._reset(this._map.getCenter(), this._map.getZoom(), e && e.hard);
                if (this._docType === 'spreadsheet' && this._annotations !== 'undefined') {
                        app.socket.sendMessage('commandvalues command=.uno:ViewAnnotationsPosition');
                }

                this._painter._viewReset(); // and calls TileSectionManager::_viewReset because this._painter = new L.TileSectionManager(this))
        },
});

**Pull request merged**